src/gluonts/mx/block/quantile_output.py

```diff
@@ -142,6 +148,64 @@ def compute_quantile_loss(
 
         return qt_loss
 
+    def compute_quantile_weights(self):
+        """
+        Compute weight of each quantile
+
+        Math:
+        Let quantiles = [q_1, ..., q_n] with quantile estimates [z_1, ..., z_n].
+        Then, approximated CRPS with the linear interpolation is
+
+        .. math::
+            :nowrap:
+
+                    CRPS = sum_{i=1}^{n-1} 0.5 * (q_{i+1} - q_{i}) * (z_{i+1} + z_{i}).
```

Comment:
There appears to be one level of indent too much here

Reply:
Fixed! Do you know how to allow flake8 to pass on '\s' characters so that we can use '\sum' here? Also I couldn't get flake8 to pass with `\begin{aligned}` and `\end{aligned}` blocks, similar to what we have in the GP code.